package.json

```diff
@@ -46,20 +49,23 @@
     "jsii-pacmak": "^1.26.0",
     "json-schema": "^0.3.0",
     "prettier": "^2.2.1",
-    "projen": "^0.16.42",
+    "projen": "^0.17.84",
```

Comment:
It seems like this may be the culprit for why the additional projen stuff was added, I would say we should stay at "^0.16.42" for now